.github/workflows/lint.yml

```
@@ -46,5 +46,5 @@ jobs:
 
       - name: Lint
         run: |
-          nvim -l scripts/format-queries.lua
+          nvim -l scripts/format-queries.lua queries
```

```suggestion
          nvim -l scripts/format-queries.lua
```

You need to rebase; I changed the script to default to current dir.

---

It also tries to format the test file, so I deliberately do that again

---

Maybe there's a better solution here :thinking:

---

Decided to `; format-ignore` pilled that file